tests/test_queries.py

```diff
@@ -16,86 +14,113 @@
 turing_paper = Paper.from_bibentry(fixtures.turing_bibentry,
                                    metadata=fixtures.turing_metadata)
 
+
 class TestAuthorFilter(unittest.TestCase):
 
     def test_fails_if_no_author(self):
         no_doe = doe_paper.deepcopy()
         no_doe.bibentry['author'] = []
-        self.assertTrue(not _check_author_match(no_doe, 'whatever'))
+        self.assertTrue(not AuthorFilter('whatever')(no_doe))
```

Comment:
AuthorFilter returns `True`/`False`.  So shouldn't this be explicit with `self.assertIs(..., False)` or `self.assertFalse`?

Reply:
Sure! That's why reviews are useful…